core/src/cost_tracking_service.rs

```diff
@@ -0,0 +1,185 @@
+//! this service receives transactions that have been successfully added
+//! to a bank from banking_stage, it therefore calculates and applies the
+//! costs of these transactions to cost_tracker.
+//! This process has some overhead, being in its own service thread is to
+//! minimize impact on main TPU threads.
```

Comment:
hmmm the name of the existing `cost_update_service.rs` and this new `cost_tracking_service.rs` don't seem to distinguish between the functionalities very well.

How about something along the lines of `block_generation_cost_tracking_service`, and `cost_governing_service", taking a page out of the book from Rent and Fees (thanks @t-nelson)

Reply:
sold! thanks @carllin and @t-nelson